source/module_relax/relax_driver.cpp

```diff
@@ -14,9 +14,13 @@ void Relax_Driver::relax_driver(ModuleESolver::ESolver* p_esolver)
     ModuleBase::TITLE("Ions", "opt_ions");
     ModuleBase::timer::tick("Ions", "opt_ions");
 
-    if (PARAM.inp.calculation == "relax" || PARAM.inp.calculation == "cell-relax")
+    if (PARAM.inp.calculation == "relax" || PARAM.inp.calculation == "cell-relax" )
     {
-        if (!PARAM.inp.relax_new)
+        if(PARAM.inp.relax_method == "bfgs_trad")
```

Comment:
some logic here have problems, please think carefully the function of 'relax_new' parameter

Reply:
@sunliang98 please help Feiyang check the codes

Reply:
@YuLiu98 please help checking the logic here